### PR TITLE
GraphQL: Add mutations for adding/removing store credit to a cart

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) adds documentation for the `applyStoreCreditToCart` and `removeStoreCreditFromCart` mutations.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/graphql/mutations/apply-store-credit.html
- https://devdocs.magento.com/guides/v2.3/graphql/mutations/remove-store-credit.html

whatsnew
- Added documentation for the [applyStoreCreditToCart](https://devdocs.magento.com/guides/v2.3/graphql/mutations/apply-store-credit.html) and [removeStoreCreditFromCart](https://devdocs.magento.com/guides/v2.3/graphql/mutations/remove-store-credit.html) mutations.

### CODE REVIEW

File: guides/v2.3/graphql/mutations/apply-store-credit.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+Store credit is an amount that the merchant applies to a customer account as a result of a refund or similar transaction. A customer can use store credit to pay for purchases.
+
+## Syntax
+
+`mutation: {applyStoreCreditToCart(input: ApplyStoreCreditToCartInput): {ApplyStoreCreditToCartOutput}}`
+
+## Example usage
+
+In the following example, the customer starts with $100 of store credit. The price of the contents of the cart is less than $100, so the applied store credit causes the grand total to be $0.
```
Comment (erikmarr) [inline guides/v2.3/graphql/mutations/apply-store-credit.md]:
```suggestion
In the following example, the customer starts with $100 of store credit. The price of the contents of the cart is less than $100, so the applied store credit causes the remaining cart balance to be $0.
```

File: guides/v2.3/graphql/reference/quote.md [hunk 2]
```diff
@@ -436,6 +436,15 @@ Attribute |  Data Type | Description
 `current_balance` | Money | Current balance remaining on the gift card
 `expiration_date` | String | Gift card expiration date
 
+#### AppliedStoreCredit object {#AppliedStoreCredit}
+
+The `AppliedStoreCredit` object can contain the following attributes:
```
Comment (erikmarr) [inline guides/v2.3/graphql/reference/quote.md]:
```suggestion
The `AppliedStoreCredit` object can contain the following attributes.
```

Comment (keharper) [reply guides/v2.3/graphql/reference/quote.md]: The colon is consistent with all the other lead-in sentences in this topic.

**Comment by meker12** [comment]
@keharper  In what's new, any bulleted items must be preceded by an opening sentence.  In this case, bullet is not required.

**Pull request merged**